sql1.txt

```diff
@@ -6,24 +6,28 @@ SELECT * FROM Products LIMIT 5;
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
 
+SELECT * FROM Сustomers ORDER BY СustomerName LIMIT 10;
```

Comment:
Your query is almost perfect! However, it seems you're using the Cyrillic `С` instead of the Latin `C`. They look the same, but the Cyrillic `С` doesn't match the table name, so it won't work. Just replace `С` with `C` both times it's used.

Reply:
I don't know how this happened, corrected!